.env.example

```diff
@@ -6,4 +6,4 @@ PG_PORT=5432
 PG_HOST=localhost
 PG_USER=''
 PG_PASSWD=''
-TEST_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6InVzZXIwIiwiZW1haWwiOiJ0ZXN0QHRlc3QuY29tIiwiYWRtaW4iOnRydWV9.X_SzIXZ-oqEL67eB-fwFqFSumuFQVAqhgsmak1JLIWo
+TEST_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6InVzZXIwIiwiZW1haWwiOiJ0ZXN0QHRlc3QuY29tIiwiYWRtaW4iOnRydWV9.7ntFOWcb6YNeW5njf6kFwL3-8mL4HQF3H2iA011s4hY

```

Comment:
Is there a reason the test token is changing? We'd generally like to keep it consistent with an "open" secret in the repository.